src/router/router.ts

```diff
@@ -1,19 +1,72 @@
-import { createRouter, createWebHistory } from 'vue-router'
-import { setupLayouts } from 'virtual:generated-layouts'
-import generatedRoutes from 'virtual:generated-pages'
+import { createRouter, createWebHistory } from "vue-router";
+import { setupLayouts } from "virtual:generated-layouts";
+import generatedRoutes from "virtual:generated-pages";
+import { useAuthStore } from "@/modules/auth/store/authStore";
 
-const routes = setupLayouts(generatedRoutes)
+const routes = setupLayouts(generatedRoutes);
+
+// const routes = [
```

Comment:
Let's delete the comments if not used
Usually commented code doesn't need to stay commented unless it's really useful